src/sentry/api/endpoints/project_environment_details.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import absolute_import
+
+from rest_framework import serializers
+from rest_framework.response import Response
+
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.api.serializers import serialize
+from sentry.models import EnvironmentProject
+
+
+class ProjectEnvironmentSerializer(serializers.Serializer):
+    is_hidden = serializers.BooleanField()
+
+
+class ProjectEnvironmentDetailsEndpoint(ProjectEndpoint):
+    def get(self, request, project, environment):
+        instance = EnvironmentProject.objects.select_related('environment').get(
```

Comment:
should you handle an `EnvironmentProject.DoesNotExist` here (and in `put`)?

Reply:
Oops, yeah, I meant to rethrow it as a `ResourceDoesNotExist` but must have rushed through it and forgot, thanks for the reminder. 😬

Reply:
Fixed with 7562bbe.